debian/control

```diff
@@ -2,33 +2,30 @@ Source: libtpms
 Maintainer: Stefan Berger <stefanb@linux.vnet.ibm.com>
 Section: libs
 Priority: optional
-Standards-Version: 3.9.7
-Build-Depends:
- autoconf,
- automake,
- autotools-dev,
```

Comment:
These dependencies are implied transitively by `debhelper` (though `dh-autoreconf`) and `libtool`.